lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -642,6 +642,9 @@ class BarTouchTooltipData with EquatableMixin {
   /// Controls showing tooltip on top or bottom, default is auto.
   final TooltipDirection direction;
 
+  /// Controls the rotation of the tooltip.
+  final double rotateAngle;
```

Comment:
May I ask you to add rotateAngle for other tooltips? (LineTouchTooltipData and ScatterTouchTooltipData).